components/usage/pkg/apiv1/billing_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package apiv1
+
+import (
+	v1 "github.com/gitpod-io/gitpod/usage-api/v1"
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestCreditSummaryForTeams(t *testing.T) {
+	teamID := uuid.New().String()
+	teamAttributionID := db.NewTeamAttributionID(teamID)
+
+	scenarios := []struct {
+		Name     string
+		Sessions []*v1.BilledSession
+		Expected map[string]int64
```

Comment:
Should we add a test with two different team attribution ids?

Reply:
Makes sense. Originally this test is just moved from the old location but it makes sense to ensure more variance.